docs/buttons/DosDontsExample.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, SvgIcon, useTheme } from '@mui/material';
+
+import { Close } from '../../packages/ui-kit-components/src/icons';
```

Comment:
We could import the icon with the reference like `import { Icon } from '@ui-kit-2022/components';` and we can use directly `<Icon.Close />` instead of the wrapping in the SvgIcon component.

Reply:
Also, were you unable to directly import `Close`?   `import { Close } from '@ui-kit-2022/components';` ?  Would be good if that could work..., might need to add an export statement to the component package `index.ts`

Reply:
I tested and I were not able to directly import `Close`. As in the component package `index.ts`, it was `export * as Icon from './icons';`, so maybe it's better just do `import { Icon } from '@ui-kit-2022/components'`

Reply:
We can change it, I think, to `export * from './icons'`, but maybe it was intended that we import `Icon` rather than the specific icon, could be confusing exporting all the icons with the components.  @iandilling-adaptive - was it intended that you cannot directly import an icon component?